pact/test/test_verify.py

```diff
@@ -138,6 +178,40 @@ def test_all_options(self):
             '--broker-username=user',
             '--broker-password=pass',
             '--broker-token=token',
+            '--pact-broker-base-url=http://localhost',
+            '--provider=provider',
+            '--publish-verification-results',
+            '--provider-app-version', '1.2.3',
+            '--verbose')
+
+    def test_all_broker_options(self):
+        self.mock_Popen.return_value.returncode = 0
+        result = self.runner.invoke(verify.main, [
+            '--pact-broker-base-url=http://localhost/broker',
+            '--consumer-version-tag=$LATEST',
```

Comment:
What is $LATEST here? Unlike docker, using the tag "latest" with the Pact Broker is confusing, as then you'd be retrieving "the latest pact with the tag 'latest'" eg. `/pacts/provider/my-provider/consumer/my-consumer/latest/latest`

Reply:
This is a good resource to read to get an idea of how tags work btw https://github.com/pact-foundation/pact_broker/wiki/Using-tags

Reply:
Gotcha - so changed it to prod as it's just a unit test but I assume some clipped git sha would also do? $LATEST in this case was a lazy AWS reference on my part.

Reply:
You'd usually specify tag names, because you, as the provider, wouldn't necessarily know the exact version of the consumer in prod. The consumer team tells the pact broker when they deploy which version is in prod by tagging the application version resource, so the provider team can just say "the latest prod version" without having to know specifics. There's a good explanation here https://docs.pact.io/pact_broker/can_i_deploy